docs/components/board/_index.md

```diff
@@ -140,7 +140,7 @@ Additionally, the nested `GPIOPin`, `Analog`, and `DigitalInterrupt` interfaces
 
 {{< readfile "/static/include/components/apis/digitalinterrupt.md" >}}
 
-### ReadAnalog
+### AnalogByName
```

Comment:
These headings should be the proto names, that's what we standardized on... https://github.com/viamrobotics/api/blob/main/component/board/v1/board_grpc.pb.go#L124
```suggestion
### ReadAnalogReader
```

Again interested to hear what Rand thinks

Reply:
It's confusing but I believe that proto is used for [Read](https://docs.viam.com/components/board/#read) -- see [client.py](https://github.com/viamrobotics/rdk/blob/f22ff365d533c441e7e697cf76dd65cf51fa9366/components/board/client.go#L158) and the pb description. Maybe we should update that heading? 
I can't find that there is a proto for this method

Reply:
So this one should be AnalogByName and the Read one below should be `ReadAnalogReader`?

Reply:
Yeah, I think so. I'll update Read too and we can see what Rand thinks when she's back from PTO

Reply:
I'm sorry that board kind of breaks the proto -> method consistency because of its design. I'm fine with these being proto methods, but just clarify to the user that you can't get to the over-the-wire message in our python and go skds without invoking the `Analog` object and calling the Read method on it after.